packages/satyrographos/satyrographos.0.0.1.5/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+maintainer: "SAKAMOTO Noriaki <mrty.ityt.pt@gmail.com>"
+authors: [
+  "SAKAMOTO Noriaki <mrty.ityt.pt@gmail.com>"
+]
+homepage: "https://github.com/na4zagin3/satyrographos"
+dev-repo: "git+https://github.com/na4zagin3/satyrographos.git"
+bug-reports: "https://github.com/na4zagin3/satyrographos/issues"
+license: "LGPL3+"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "cmdliner"
+  "core"
+  "dune" {build}
+  "fileutils"
+  "json-derivers"
+  "ppx_deriving" {build}
+  "ppx_inline_test" {build}
+  "ppx_jane" {build}
+  (* "satysfi" {>= "0.0.3" & < "0.0.4"} *)
```

Comment:
Why is this commented out?

Reply:
Because Satyrographos has stopped depending on [SATySFi](https://github.com/gfngfn/SATySFi). I should have removed it, but I didn’t. Fixed it.

Thank you for your comments!